tests/Microsoft.Identity.Test.Integration.netcore/SeleniumTests/DeviceCodeFlowIntegrationTest.cs

```diff
@@ -83,6 +83,7 @@ public async Task ArlingtonDeviceCodeFlowAdfsTestAsync()
         [TestMethod]
         [Timeout(2 * 60 * 1000)] // 2 min timeout
         [TestCategory(TestCategories.MSA)]
+        [Ignore]
```

Comment:
Can you please push a separate PR for this?

Reply:
this is dead code, removing